CHANGELOG.md

```diff
@@ -14,6 +14,11 @@ Changes are grouped as follows
 - `Fixed` for any bug fixes.
 - `Security` in case of vulnerabilities.
 
+## [2.26.0] - 2021-07-08
```

Comment:
```suggestion
## [UNRELEASED]
```
Or bump the version number in cognite/client/_version.py, which will trigger a new release to pypi.